src/renderer/components/badge/badge.tsx

```diff
@@ -3,21 +3,43 @@ import "./badge.scss"
 import React from "react";
 import { cssNames } from "../../utils/cssNames";
 import { TooltipDecoratorProps, withTooltip } from "../tooltip";
+import { observer } from "mobx-react";
+import { autobind } from "../../utils";
+import { observable } from "mobx";
 
 export interface BadgeProps extends React.HTMLAttributes<any>, TooltipDecoratorProps {
   small?: boolean;
   label?: React.ReactNode;
+  isExpanded?: boolean; // by default: false (minimized)
```

Comment:
Why is this a prop, what has the parent go to do with expanding the badge?

Reply:
`isExpanded={true}` will allow to show all contents from the Badge always (forced expanded state)

Reply:
Okay sounds good